pages/dmp_supporting_info/support_09_responsibilities_resources.md

```diff
@@ -23,17 +23,46 @@ turing:
 
 {% include callout.html type="tip" content="
 **Why is this topic important**\\
->> What is in for you?\\
->> Short & concise\\
->> Add lines as needed\\
+>> Defining responsibilities, project involvement and cost related to research data management is highly important when planning and conducting research projects\\
+>> Relevant estimates of costs for research data management should be included when applying for financing, and this chapter will help you to develop relevant estimates\\
 " %}
 
 ## About this chapter
 
-General information
+In this chapter, you are asked to define responsibilities, who will be involded in the project, and the cost related to research data management. Remeber that additional expertice can be needed when covering the areas of responsibility in the project. Also remeber that cost can be relevant both for facilities, equipment, hardware, software, license costs or other recurring fees, preservation and archiving, costs for preparing data for publication and more. The questions in this chapter cover such aspects of research data management.       
 
 ## Question-specific guidance
 
-### Question 1
+### Have data management responsibilities in the project been assigned?
+Please specify. 
```

Comment:
```suggestion
Being specific about data management responsibilities in a project is recommended. Contributors and their roles should be specified in the chapter 'About the Data Management Plan'. In educational settings, the supervisor(s) should be specified as they have formal responsibility for data management.
```

Reply:
We should consider if this one should be changed to an 'item select' question.

Reply:
Yes, agree